src/modules/IscsiClientLib.rb

```diff
@@ -220,7 +220,7 @@ def nodeInfoToMap(stdout)
         key, val = row.split("=")
 
         key = key.to_s.strip
-        retval[key] = val.to_s.strip if !key.empty?
+        retval[key] = val.to_s.strip if !key.empty? && !key.match?(/^#/)
```

Comment:
`!key.start_with?("#")` is a simpler equivalent.

Reply:
`/^\s*#/` would probably be the semantically correct check. I'd leave it as is, though.